### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by sayalikotkar**
I just updated the readme with the current information about our project & moved the collabathon content to another location (saved it in a new issue)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,59 +1,23 @@
...
-# Contact info
-
-Contact our Product Manager [Angela Zhang](https://app.slack.com/team/U01CDAWS3HP) with any inquiries, questions or concerns.
-
-
-### Licensing
 
-Include details about the project's open source status.
+- [Join the HackforLA slack workspace](https://hackforla.org/slack)
+- Join the slack channel [#climate-collabathon](https://hackforla.slack.com/archives/C01C2P7D413) and say hi! Tell us what your skillset is and how you're looking to contribute.
```
Comment (angle-zhang) [inline README.md:19]: we need to change the name of the channel still so maybe reach out to bonnie about that first?

Comment (sayalikotkar) [reply README.md]: @angle-zhang I updated the name of the slack channel. Please review again

Comment (angle-zhang) [inline README.md]: change this to the name of our current slack channel

Comment (sayalikotkar) [reply README.md]: It's been changed

**Review by angle-zhang — APPROVED** [review]

**Pull request merged**